### DISCUSSION TITLE
[Achievement] Badge shown even if a contribution to a private repository

### DISCUSSION THREAD

**Discussion by Xwilarg** [community/community#39083] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hey,

So in my settings I have "Include private contributions on my profile" unchecked however when I go on my profile in private navigation I see that I have the "Extraordinary Pair" badge with the repository related being "inaccessible · Coauthored with an unknown user"

Shouldn't the badge just not being shown in that case?

**Comment by david-engelmann** [comment]
You did accomplish that, but you can hide the badge from the achievements screen
